src/static/css/pages/beta.less

```diff
@@ -77,6 +72,7 @@
     .respond-to-min(801px, {
         // At this width, the wrapper gains `position: relative`,
         // so the button position has to be adjusted accordingly.
+        // This number depends on something in cf-layout
```

Comment:
It turns the media query into `em`s, but I was asking about the comment you made in reference to `top: -12px`. Never really liked magic numbers so I wasn't sure if there's a way we could better future proof it.